README.md

```diff
@@ -55,3 +55,16 @@ This project uses [`LocalStack`](https://github.com/localstack/localstack) for i
 ```
 $ aws --endpoint=http://localhost:4566 route53 list-hosted-zones-by-name
 ```
+
+## AWS account setup
+
+To run the integration and acceptance tests against an AWS account you need to create an IAM role. To do that, first source the `aws-resolver-rules-operator-test-secrets.sh` file from LastPass and then run the following commands:
+
+```shell
+aws iam create-policy --policy-name tests-aws-network-topology-operator --policy-document file://tests/assets/test-role-policy.json
```

Comment:
Oh another thing. Can you mention that the aws cli needs to be configured to target the AWS account found in the `WC_AWS_ACCOUNT` variable of the file?

Reply:
Done

Reply:
Does the role need to exist on both MC and WC account? I think I'd mention in which one is required.

Reply:
I didn't bother splitting up the permissions between the MC and WC account, but they both need permissions. For RAM in particular you need to be able to at least list the resource in the recipient account.